index.html

```diff
@@ -0,0 +1,225 @@
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.10.4/font/bootstrap-icons.css"
+    />
+    <link rel="stylesheet" href="styles/style.css" />
+    <title>AI Brains</title>
+  </head>
+  <body>
+    <header class="hidden-desktop container-fluid p-0">
+      <div class="d-flex p-2">
+        <ul
+          class="container d-flex justify-content-end px-5 fs-6 m-0 gap-3 text-white"
+        >
+          <li><i class="bi bi-facebook"></i></li>
+          <li><i class="bi bi-twitter"></i></li>
+          <li>English</li>
+          <li>My Page</li>
+          <li>Log out</li>
+        </ul>
+      </div>
+      <div class="container-fluid bg-white d-flex justify-content-center">
+        <div class="container d-flex justify-content-between p-4 fs-6 m-0">
+          <a href="index.html"
+            ><img src="img/AIBrain.ico" alt="AI brain logo" width="30px" /> AI
+            <span>B</span>rains</a
+          >
+          <ul class="d-flex fs-6 align-items-center m-0 gap-3 p-0">
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <li>
+              <a href="#" class="border border-4 border-danger p-2 text-danger"
+                >AI Campaign</a
+              >
+            </li>
+          </ul>
+        </div>
+      </div>
+    </header>
+    <!-- Main content -->
+    <main class="container-fluid p-0">
+      <!-- Intro section -->
+      <!-- Menu mobile -->
+      <div id="menu" class="hidden-mobile flex-column">
+        <div class="menu-mobile flex-column">
+          <button class="close-btn"><i class="bi bi-x-lg"></i></button>
+          <ul class="menu-options">
+            <li><a href="index.html">Home</a></li>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Program</a></li>
+            <li><a href="#">Join</a></li>
+            <li><a href="#">Sponsor</a></li>
+            <li><a href="#">News</a></li>
+            <li>
+              <a href="#" class="border border-4 border-danger p-2 text-danger"
+                >AI Campaign</a
+              >
+            </li>
+          </ul>
+        </div>
+      </div>
+      <img
+        src="img/bg-intro-min.png"
+        alt="AI background image"
+        class="bg-image"
+      />
+      <section class="intro-container d-flex flex-column gap-3 p-3 p-lg-5">
+        <button class="hidden-mobile menu-btn btn align-self-start p-2">
+          <img src="img/btn-menu.png" alt="menu mobile" width="24px" />
+        </button>
+        <span class="fs-3 mt-5 title">"Hello! Human world"</span>
+        <h1 class="mb-3 title fw-bold">AI brains Global Summit 2023</h1>
+        <blockquote
+          class="col-sm-4 col-md-6 col-lg-6 border border-3 border-white fs-6 p-3"
+        >
+          "The best way to predict the future is to invent it." - Alan Kay
```

Comment:
- [OPTIONAL ] I highly recommend to add more description here to make your project more good for user expereince (UX) so it would be great if you add it as well 🔥💯